apps/dashboard/src/routes/+layout.ts

```diff
@@ -2,11 +2,12 @@ import { PUBLIC_SUPABASE_ANON_KEY, PUBLIC_SUPABASE_URL } from '$env/static/publi
 import { createSupabaseLoadClient } from '@supabase/auth-helpers-sveltekit';
 import { redirect } from '@sveltejs/kit';
 import type { LayoutLoad } from './$types';
+import type { Database } from '../types/database.types';
 
 export const load: LayoutLoad = async ({ fetch, data, depends }) => {
 	depends('supabase:auth');
 
-	const supabase = createSupabaseLoadClient({
+	const supabase = createSupabaseLoadClient<Database>({
```

Comment:
Need to remember to add `<Database>` whenever we create a client. 

Can perhaps put this in a helper function in lib?